docs/architecture/specs/marketplace_api/README.md

```diff
@@ -0,0 +1,265 @@
+editor:         Aitor Argomaniz <aitor@nevermined.io>
+contributors:   
+```
+
+Table of Contents
+=================
+
+   * [MKT-API SPEC: Nevermined Marketplace API](#mkt-api-spec-nevermined-marketplace-api)
+   * [Table of Contents](#table-of-contents)
+      * [Motivation](#motivation)
+      * [Architecture](#architecture)
+      * [Modules](#modules)
+         * [Assets Management](#assets-management)
+         * [Assets Search](#assets-search)
+         * [Users Profiles](#users-profiles)
+         * [Secondary Market](#secondary-market)
+         * [NFTs Metadata](#nfts-metadata)
+         * [User Reviews](#user-reviews)
+         * [Bookmarks](#bookmarks)
+         * [Marketplace Management](#marketplace-management)
+         * [Multimedia](#multimedia)
+         * [API Access Control](#api-access-control)
+            * [Authentication](#authentication)
+            * [Authorization](#authorization)
+         * [Storage](#storage)
+      * [Links](#links)
+
+
+---
+
+This SPEC introduces a new API that helps to build Marketplace environments on top of the Nevermined core protocol.
+
+## Motivation
+
+With this Specification we want to build the foundations of an API that can be used as reference to implement 
+APIs exposing typical Marketplace functionalities.
+
+Most of the Marketplaces (decentralized or not) expose some common functionalities around the assets they manage and 
+their users. The intention of this API is to normalize a common set of this functionalities, allowing to build 
+implementations providing this common functionalities.
+
+With that objective in mind, the specification of this API should detail the following capabilities:
+
+* Assets Management - Creation, Update, Deleting, etc. This is based in the existing [Metadata Spec](../metadata/README.md)
+* Assets Search - Allowing the search and filtering of assets
+* Users Profile - Allowing to create, update and delete users
+* Secondary Market - Registering the negotiation of parties during a secondary market purchase
+* NFTs Metadata - Recording the metadata associated to a NFT 
+* User Reviews - Allowing users to comment and rate marketplace contents
+* Bookmarks - Allowing users to save bookmarks about different assets
+
+
+## Architecture
+
+## Modules
+
+The API will provide the following functionalities.
+
+### Assets Management
+
+The Assets in a Nevermined ecosystem are represented via DDOs that are resolved via a Decentralized Identifier (DID). While the DIDs are 
+unique identifiers stored on-chain, the metadata of an asset is represented by a DDO recorded in an external data storage (centralized or decentralized).
+This DDO includes all the additional information describing an asset and the services attached to that asset (access, computation, etc.).
+
+This module of the API allows the management of the assets information. The main functionalities provided are the following:
+
+* Create a DDO of a new asset - `POST /api/v1/metadata/assets/ddo`
```

Comment:
is this creating the DDO with publishing on-chain? if so, how do we handle signing of transactions?

Reply:
Nope, this only creates the Metadata representation. This endpoint is a copy of the same one already existing in the metadata-api